docs/conf.py

```diff
@@ -7,13 +7,11 @@
 
 import os
 import sys
-import shutil
-import sphinx_gallery.gen_gallery
 import skl2onnx
-import onnxruntime
-import sphinx_skl2onnx_extension
 import sphinx_modern_theme_modified
 
+sys.path.append(os.path.abspath('exts'))
```

Comment:
This is general approach to add custom extensions.
http://www.sphinx-doc.org/en/master/usage/extensions/index.html#where-to-put-your-own-extensions